Client/InfluxDBClientOptions.cs

```diff
@@ -23,29 +24,223 @@ public class InfluxDBClientOptions
             RegexOptions.CultureInvariant |
             RegexOptions.RightToLeft);
 
-        public string Url { get; }
-        public LogLevel LogLevel { get; }
+        private char[] _token;
+        private string _url;
+        private TimeSpan _timeout;
+        private LogLevel _logLevel;
+        private string _username;
+        private char[] _password;
```

Comment:
You can change `_token` to `string` for better future maintenance and simplify the code. This breaking change is only in our internal API, but you have to add a note about this into `CHANGELOG.md`.
```suggestion
        private string _password;
```